workspace/bus.ts

```diff
@@ -0,0 +1,5 @@
+export class Bus {
+    constructor(public line: string, public timeToArrival: string, public destination: string) {
```

Comment:
I really like this way of setting up the constructor! Avoids a lot of unnecessary [boilerplate](https://en.wikipedia.org/wiki/Boilerplate_code)